docs/en/source/_build/html/_sources/onepageTutorial.rst.txt

```diff
@@ -0,0 +1,327 @@
+
+- **Select your installation folder**
+
+ Specify the folder where you want Eclipse to be installed. The default folder will be in your User directory.Select the ‘Install’ button to begin the installation.
+
+- **Launch Eclipse**
+
+ Once the installation is complete you can now launch Eclipse. The Eclipse Installer has done it's work. Happy coding.
+
+.. figure:: figures/.png
+
+*****************************************
+JDK Setup
+*****************************************
+
+- Start the JRE installation and hit the “Change destination folder” checkbox, then click 'Install.'
+
+
+- Change the installation directory to any path without spaces in the folder name. E.g. C:\Java\jre1.8.0_xx\. (By default it will be C:\Program Files\Java\jre1.8.0_xx), then click 'Next.'
+
+
+- Now open the system environment variables dialogue by opening Control Panel -> System -> Advanced system settings -> Environment Variables.
+
+
+- Hit the New User Variable button in the User variables section, then type JAVA_HOME in Variable name and give your jre path in the Variable value. It should look like the below image:
+
+.. figure:: figures/.png
+
+ (Java path and version may change according to the version of Kafka you are using)
+
+- Now click OK.
+
+- Search for a Path variable in the “System Variable” section in the “Environment Variables” dialogue box you just opened.
+
+- Edit the path and type “;%JAVA_HOME%\bin” at the end of the text already written there, just like the image below:
+
+.. figure:: figures/.png
+
+
+- To confirm the Java installation, just open cmd and type “java –version.” You should be able to see the version of Java you just installed.
+
+.. figure:: figures/.png
+
+If your command prompt somewhat looks like the image above, you are good to go. Otherwise, you need to recheck whether your setup version matches the correct OS architecture (x86, x64), or if the environment variables path is correct.
+
+*****************************************
+ZooKeeper Installation
+*****************************************
+
+
+1. Go to your ZooKeeper config directory. For me its C:\zookeeper-3.4.7\conf
+2. Rename file “zoo_sample.cfg” to “zoo.cfg”
+3. Open zoo.cfg in any text editor, like Notepad; I prefer Notepad++.
+4. Find and edit dataDir=/tmp/zookeeper to :\zookeeper-3.4.7\data  
+5. Add an entry in the System Environment Variables as we did for Java.
+
+ a. Add ZOOKEEPER_HOME = C:\zookeeper-3.4.7 to the System Variables.
+ b. Edit the System Variable named “Path” and add ;%ZOOKEEPER_HOME%\bin; 
+
+6. You can change the default Zookeeper port in zoo.cfg file (Default port 2181).
+7. Run ZooKeeper by opening a new cmd and type zkserver.
+8. You will see the command prompt with some details, like the image below:
+
+.. figure:: figures/.png
+
+
+*****************************************
+Setting Up Kafka
+*****************************************
+
+1. Go to your Kafka config directory. For me its **C:\kafka_2.11-0.9.0.0\config**
```

Comment:
This is all quite windows specific. Please add remarks when something is OS specific.
Generally i would a avoid formulations such as For me it's. Rather use E.g. or For example.

Reply:
we have added new section ,that's for the linux .Please add the steps for linux bases installation if you find some time. And we have no mac machines so at the moment we are unable to add the mac bases installation steps.